fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -210,11 +210,20 @@ protected Response getContent(final String rangeValue,
                 prefer.getReturn().addResponseHeaders(servletResponse);
             }
         }
-        servletResponse.addHeader("Vary", "Accept, Range, Accept-Encoding, Accept-Language");
 
+        addVaryHeader(resource());
         return ok(outputStream).build();
     }
 
+
+    protected void addVaryHeader(final FedoraResource resource) {
+        final StringBuilder varyValues = new StringBuilder("Accept, Range, Accept-Encoding, Accept-Language");
```

Comment:
I realize that you did not establish the approach of including a series of `Vary` headers as a single String, but I would suggest looping through these values and adding them individually.
For example, the current approach produces:
```
Vary: Prefer
Vary: Accept, Range, Accept-Encoding, Accept-Language, Accept-Datetime
```
...which I think is cleaner and more useful to the client as:
```
Vary: Prefer
Vary: Accept
Vary: Range
Vary: Accept-Encoding
Vary: Accept-Language
Vary: Accept-Datetime
```